### ISSUE TITLE
not necessarily an issue with this (I think)

### ISSUE THREAD

**Issue by LevelNaomi** [picotorrent/picotorrent#1277]
basically, let's say that someone is seeding from another torrent app, and i try to download that file with picotorrent, will it work?
if anyone needs a better explanation i could try, i'm not the best at explaining things

**Comment by cyberhardt** [comment]
yes, all torrent clients will seed and download from each other because they use the same protocol.
